.github/workflows/composite/maximize-build-space/action.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: maximize-build-space
+description: Maximize build space on github runner
+
+runs:
+  using: composite
+  steps:
+    - name: Maximize build space
+      shell: bash
+      run: |
+        echo "Available storage before:"
+        df -h
+        echo "Removing /usr/share/dotnet, /usr/local/lib/android, /opt/ghc"
```

Comment:
Is that not redundant documentation?

Reply:
The `rm` calls take ~1min which does not produce any logging, so I think it is justified to print one line to give feedback.